packages/test-commerce-sdk-react/app/components/Json.tsx

```diff
@@ -4,13 +4,42 @@
  * SPDX-License-Identifier: BSD-3-Clause
  * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
  */
-import React from 'react'
+import React, {useState} from 'react'
 
-const Json = React.memo(({data}: {data: any}) => (
-    <div>
-        <pre>{JSON.stringify(data, null, 2)}</pre>
-    </div>
-))
+const Json = ({data}: {data: any}) => {
+    const [expanded, setExpanded] = useState(false)
+
+    const style = {
+        body: {
+            position: 'relative',
+            height: expanded ? 'inherit' : '100px',
+            overflow: 'hidden'
+        },
+        button: {
+            position: 'absolute',
+            right: 0,
+            paddingRight: 10
+        },
+        shadow: {
+            position: 'absolute',
+            bottom: 0,
+            left: 0,
+            width: '100%',
+            height: 10,
+            background:
+                'linear-gradient(to bottom, rgba(137,255,241,0) 0%,rgba(255,255,255,1) 100%)'
+        }
+    }
+    return (
+        <div style={style.body}>
+            <a style={style.button} onClick={() => setExpanded(!expanded)}>
+                {expanded ? '[-]' : '[+]'}
+            </a>
```

Comment:
I was tired of scrolling through all the response objects. This made things less infuriating. 🤣